packages/node_modules/@webex/plugin-meetings/src/metrics/config.js

```diff
@@ -286,8 +285,31 @@ export const errorCodes = {
   NOT_ANNOUNCEMENT_SPACE: 1403015,
   USER_NOT_MODERATOR_IN_ANNOUNCEMENT_SPACE: 1403016,
   TEMP_ID_ALREADY_EXISTS: 1409001,
-  PARENT_ACTIVITY_ID_NOT_FOUND_OR_INVALID: 14000015
+  PARENT_ACTIVITY_ID_NOT_FOUND_OR_INVALID: 14000015,
+
+  // locus error codes: https://sqbu-github.cisco.com/WebExSquared/locus/blob/master/server/src/main/resources/locus-error-codes.properties
+  2502007: 'SERVICE_UNAVAILABLE',
```

Comment:
does the server send all of these as un avaliable ?

Reply:
Will check and confirm w/ call analyzer